### PR TITLE
Implementing Header editor

### PR DESCRIPTION
**Pull request by byeolbit**
Implementing Header editor

ISSUE = #25 

Fix date calculation for URI
ISSUE = #16 

Progress : 
- [x] : Implement basic structure of component
- [ ] : Style sheet
- [ ] : Save feature

### CODE REVIEW

File: src/app/editor/sidebar/file-browser/headerVariable.ts [hunk 1]
```diff
@@ -1,8 +1,8 @@
 export class HeaderVariable {
   key: string;
-  value: string;
+  value: any;
```
Comment (haeungun) [inline src/app/editor/sidebar/file-browser/headerVariable.ts]: Why did you change this to 'any'?

Comment (byeolbit) [reply src/app/editor/sidebar/file-browser/headerVariable.ts]: Line 293~295 on src/app/editor/sidebar/file-browser/file-browser.component.ts
```js
         if (variable[0] === 'date') {
            variable[1] = new Date(variable[1]);
          }
```
Good point. It's not a good code.
I changed *date* key has `Date` type *value* to fix Issue #16 
But I didn't fix the constructor of HeaderVariable to fast fix for demo.
This have to be changed.
